packages/opera.rb

```diff
@@ -3,12 +3,28 @@
 class Opera < Package
   description "Opera isn't just a browser. It's an awesome browser."
   homepage 'https://www.opera.com/'
-  version '74.0.3911.154'
+  version '74.0.3911.160'
   compatibility 'x86_64'
+  
+  case ARCH
+  when 'x86_64'
+    version '74.0.3911.160'
+  when 'i686'
+    version '45.0.2552.898'
+  end
+  
+  binary_url ({
+      i686: "https://deb.opera.com/opera-stable/pool/non-free/o/opera-stable/opera-stable_#{version}_i386.deb",
+    x86_64: "https://get.geo.opera.com/pub/opera/desktop/74.0.3911.160/linux/opera-stable_#{version}_amd64.deb"
+  })
+  binary_sha256 ({
+      i686: '0fd8497f914efc47eb1ef1c7ce052487e3ca0c4c5474f01ad7e721f6ea08fea8',
+    x86_64: `curl -Ls https://get.geo.opera.com/pub/opera/desktop/#{version}/linux/opera-stable_#{version}_amd64.deb.sha256sum`.chomp
+  })
```

Comment:
I was unable to get sommelier working on my i686 Chromebook so not sure if this version is needed.

Reply:
source_url, not binary_url?

Reply:
Sommelier might have a dependency on a newer version of the exo wayland server. Do wayland apps work on i686?